lib/nightmare.js

```diff
@@ -80,8 +80,9 @@ function Nightmare(options) {
 
   process.setMaxListeners(Infinity);
   process.on('uncaughtException', function(err) {
-    console.error(err.stack);
+    debug(err.stack);
```

Comment:
Ah! Totally missed that. I wonder if `process.exit(1)` (I gather `1` is correct based on https://nodejs.org/api/process.html#process_exit_codes) instead of re-throwing would work? Then we’d be in control of logging the correct stack.